File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.py
class Solution(object):
    def finalValueAfterOperations(self, operations):
        """
        :type operations: List[str]
        :rtype: int
        """
        cnt=0
        for i in operations:
            if '+' in i:
                cnt+=1
            else:
                cnt-=1
        return cnt

File: 2011-final-value-of-variable-after-performing-operations/2011-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int finalAnsAfterOpe=0;
        for (String operation : operations) {
            if (operation.contains("+"))finalAnsAfterOpe++;
            else finalAnsAfterOpe--;
            
        }
        return finalAnsAfterOpe;
    }
}